cairo-sys-rs/src/enums.rs

```diff
@@ -328,6 +328,30 @@ pub enum RegionOverlap {
     Part,
 }
 
+#[cfg(any(feature = "pdf", feature = "dox"))]
+#[repr(C)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum PdfVersion {
+    _1_4,
+    _1_5,
```

Comment:
See https://github.com/gtk-rs/gir/issues/480

Reply:
What do I need to do exactly here? Kind of unclear from the issue.

Should I just add an `Unknown` variant?

Reply:
In the -sys crate you use an integer and constants, in the non-sys crate you have a real enum with the Unknown variant

Reply:
Is this still the case?

Reply:
Yes